pallets/gear/src/manager/task.rs

```diff
@@ -62,7 +61,10 @@ pub fn get_maximum_task_gas<T: Config>(task: &ScheduledTask<T::AccountId>) -> Ga
         RemoveGasReservation(_, _) => {
             <T as Config>::WeightInfo::tasks_remove_gas_reservation().ref_time()
         }
-        RemoveResumeSession(_) => 0,
+        #[allow(deprecated)]
+        RemoveResumeSession(_) => {
+            unreachable!("deprecated")
```

Comment:
return 0 here. check out Self::PauseProgram
it should work the same, because pause_program is deprecated